modules/ding_event/ding_event.pages_default.inc

```diff
@@ -641,6 +645,8 @@ events
   $display->cache = array();
   $display->title = 'Events';
   $display->uuid = 'd49a3b8f-8e68-4ec6-8382-b2d01945b356';
+  $display->storage_type = 'page_manager';
```

Comment:
How is this related to the case?

Reply:
This is required by panels module in order to fix some access permissions issues, introduced in version 7.x-3.6, when on pane editing administrators had "access denied" error.